rustup.hcl

```diff
@@ -103,7 +103,26 @@ on "unpack" {
   }
 }
 
+
 version "1.25.1" {
+  binaries = [
+    "cargo",
+    "cargo-clippy",
+    "cargo-fmt",
+    "cargo-miri",
+    "clippy-driver",
+    "rls",
+    "rust-gdb",
+    "rust-gdbgui",
+    "rust-lldb",
+    "rustc",
+    "rustdoc",
+    "rustfmt",
+    "rustup",
+  ]
+}
+
+version "1.25.2" {
```

Comment:
The rust-analyzer linking will need to be moved in here too.

I also think that binaries needs to be defined in each version block separately, not inherited.

Reply:
Oh or maybe it doesn't need to be moved in here, it will link it, just not expose it, so I guess that's okay?